projects/ngrx.io/content/guide/router-store/selectors.md

```diff
@@ -33,22 +30,85 @@ export const {
   selectRouteParam,     // factory function to select a route param
   selectRouteData,      // select the current route data
   selectUrl,            // select the current url
-} = fromRouter.getSelectors(selectRouter);
+} = getSelectors(selectRouter);
+```
+
+`car.state.ts`
```

Comment:
This should be split into two files: `car.reducer.ts` and `car.selectors.ts`. Using a single `.state` file isn't a common pattern we promote so it might be confusing with the rest of the docs